internal/operands/common-templates/constants.go

```diff
@@ -12,4 +12,6 @@ const (
 	TemplateFlavorLabelPrefix    = "flavor.template.kubevirt.io/"
 	TemplateWorkloadLabelPrefix  = "workload.template.kubevirt.io/"
 	TemplateDeprecatedAnnotation = "template.kubevirt.io/deprecated"
+
```

Comment:
nit: remove empty line

Reply:
I wanted to separate it from the other constants, because it is a parameter name, not a label or annotation.